src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -793,7 +814,7 @@ public static void severe(Logger logger, Throwable e, String message){
         if(e != null) {
             StringWriter sw = new StringWriter();
             e.printStackTrace(new PrintWriter(sw));
-            logger.finer(message + ": " + sw.toString());
```

Comment:
This change should be a good addition but we might want to do some testing when there are severe errors. We might want to change this to info or debug.